src/siteConfig.js

```diff
@@ -16,4 +16,5 @@ exports.siteConfig = {
     apiKey: 'e8451218980a351815563de007648b00',
     indexName: 'beta-react',
   },
+  googleAnalyticsTagId: 'G-B1E83PJ3RT',
```

Comment:
should we remove the already existing google analytics? we were using [`ga-lite`](https://github.com/reactjs/react.dev/pull/4355)

Reply:
Yes but maybe in another PR to keep the changes separate

Reply:
Cleanup here https://github.com/reactjs/react.dev/pull/6366